### PR TITLE
ports superficial healing from beestation with some modifications (port)

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
"adds Superficial healing, a level 6 symptom that heals you of brute, toxin, and burn damage, if you have fifteen or less, thirty with a threshold met. causes you to take 100 points of cloning damage  (200 points of cumulative healing)"

original pr: https://github.com/BeeStation/BeeStation-Hornet/pull/1381

#### Changelog

:cl:  
rscadd: ports a symptom from bee
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
fuck

**Comment by fluffe9911** [comment]
Isnt getting disfigured a good thing

**Comment by boodaliboo** [comment]
> Isnt getting disfigured a good thing

debatebly

**Comment by boodaliboo** [comment]
how do i fix this
image

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 1]
```diff
@@ -481,3 +481,56 @@
...
+
+	if(M.getToxLoss() && M.getToxLoss() <= threshhold)
+		M.adjustToxLoss(-power)
+		healed = TRUE
+
+	if(healed)
+		if(prob(10))
+			to_chat(M, "<span class='notice'>Your wounds heal, granting you a new scar</span>")
+		if(scarcounter >= 200)
+			adjustCloneLoss(100)
```
Comment (jjpark-kb) [inline code/datums/diseases/advance/symptoms/heal.dm]:
```suggestion
			M.adjustCloneLoss(100)
```
I think this is what it's supposed to be.

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 2]
```diff
@@ -481,3 +481,56 @@
...
+	resistance = -2
+	stage_speed = -2
+	transmittable = 1
+	level = 6
+	passive_message = "<span class='notice'>Your skin tingles.</span>"
+	var/threshhold = 15
+	var/scarcounter = 0
+	
+	threshold_descs = list(
+		"Stage Speed " = "Doubles healing speed.",
```
Comment (alexkar598) [inline code/datums/diseases/advance/symptoms/heal.dm:498]: Stage speed?

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 3]
```diff
@@ -481,3 +481,56 @@
...
+	if(M.getToxLoss() && M.getToxLoss() <= threshhold)
+		M.adjustToxLoss(-power)
+		healed = TRUE
+
+	if(healed)
+		if(prob(10))
+			to_chat(M, "<span class='notice'>Your wounds heal, granting you a new scar</span>")
+		if(scarcounter >= 200)
+			M.adjustCloneLoss(100)
+			M.visible_message("<span class='warning'>[M]'s your body starts to break down </span>", "<span class='userdanger'>Your scars have made you genetically unstable.</span>")
```
Comment (alexkar598) [inline code/datums/diseases/advance/symptoms/heal.dm:531]: What prevents this from running over and over again essentially killing the person instantly?

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/datums/diseases/advance/symptoms/heal.dm (hunk 3 above)
Comment (boodaliboo) [reply code/datums/diseases/advance/symptoms/heal.dm:531]: you can only get over that point once

File: code/datums/diseases/advance/symptoms/heal.dm (hunk 2 above)
Comment (boodaliboo) [reply code/datums/diseases/advance/symptoms/heal.dm:498]: have you never done viro?

**Review by JamieD1 — APPROVED** [review]

File: code/datums/diseases/advance/symptoms/heal.dm (hunk 2 above)
Comment (alexkar598) [reply code/datums/diseases/advance/symptoms/heal.dm:498]: It doesnt specify what speed

File: code/datums/diseases/advance/symptoms/heal.dm (hunk 3 above)
Comment (alexkar598) [reply code/datums/diseases/advance/symptoms/heal.dm:531]: After testing, I was right, this just murders you when get over the scar counter

**Review by alexkar598 — CHANGES_REQUESTED** [review]
.

**Review by Hopekz — CHANGES_REQUESTED** [review]
make the scarcounter over 200 damage check not run in a loop like Alex pointed out.
Not sure if it was clear with his request comment so making this.

**Pull request closed without merging**